modules/core/example/js/eth/create-wallet-address.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Create a new receive address on an existing multi-sig ethereum wallet at BitGo.
+ *
+ * Copyright 2021, BitGo, Inc.  All Rights Reserved.
+ */
+ const BitGoJS = require('bitgo');
+ const Promise = require('bluebird');
+
+ // change this to env: 'production' when you are ready for production
+ const bitgo = new BitGoJS.BitGo({ env: 'test' });
+
+ // Change coin to 'eth' when working with production
+ const coin = 'teth';
+ // TODO: set your access token here
+ const accessToken = '';
```

Comment:
environment variable can work here as well

Reply:
All the other examples follow the same pattern, so to keep it consistent throughout, I just kept it like this